.changeset/few-plums-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-common': patch
+---
+
+Updated Git to allow for bare clone
```

Comment:
```suggestion
Adds `depth` and `noCheckout` options to Git clone
```

Reply:
Some text about adding `log` would be good to :)

Reply:
Totally forgot to add details on that, I've updated, should be good now :)